Has `--deltas` option been removed?

Since 1.0.2 update Bodhi tests started to fail due to
```
E           subprocess.CalledProcessError: Command '['createrepo_c', '--groupfile', 'comps.xml', '--deltas', '--xz', '--database', '--quiet', '/tmp/tmp23ga5f6cbodhi/f17-updates-testing/compose/Everything/i386/os/f17-updates-testing/i386']' returned non-zero exit status 1.
/usr/lib64/python3.12/subprocess.py:413: CalledProcessError
---------------------------- Captured stderr setup -----------------------------
Argument parsing failed: Unknown option --deltas
```

I can't find any documentation about that option been removed.

--- a/createrepo_c.spec
+++ b/createrepo_c.spec
@@ -4,7 +4,9 @@
 
 %global bash_completion %{_datadir}/bash-completion/completions/*
 
-%if ( 0%{?rhel} && ( 0%{?rhel} <= 7 || 0%{?rhel} >= 9 ) ) || ( 0%{?fedora} && 0%{?fedora} >= 39 )
+# Fedora infrastructure needs it for producing Fedora ≤ 39 and EPEL ≤ 7 repositories
+# See https://github.com/rpm-software-management/createrepo_c/issues/398
+%if ( 0%{?rhel} && ( 0%{?rhel} <= 7 || 0%{?rhel} >= 9 ) ) || ( 0%{?fedora} && 0%{?fedora} >= 45 )
 %bcond_with drpm
 %else
 %bcond_without drpm

